### DISCUSSION TITLE
[Q&A] 如何修改默认模型？

### DISCUSSION THREAD

**Discussion by huangyihao2003** [lobehub/lobehub#406] [General Question]
### 🧐 问题描述 | Proposed Solution

如何修改默认模型？

**Comment by arvinxx** [comment]
你是指在哪个层面的修改？是用户自己设定，还是指部署？

如果是用户自己的设定，在 设置 -> 默认助手-> 模型设置 里，将模型改为 你要的模型即可

**Reply by huangyihao2003** [reply]
部署，在进入网站的时候就显示默认模型为gpt4

**Reply by arvinxx** [reply]
目前还没有这个配置。提个 feature request 的 issue 吧，我有空加一个~
